app/Resources/views/search/search.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html.twig' %}
```

Comment:
I'm talking o-f the file here, not the link, commenting the first line is trivial.